lib/board.rb

```diff
@@ -69,4 +69,10 @@ def valid_vertical_coordinates?(coordinates)
   def can_place_ship?(coordinates)
     valid_horizontal_coordinates?(coordinates) || valid_vertical_coordinates?(coordinates)
   end
+
+  def place(ship, coordinates)
+    coordinates.each do |coordinate|
+        @cells[coordinate].place_ship(ship)
```

Comment:
note to myself:
hash[key] access the value using this (the key is the coordinate)
in this method, ruby is grabbing the value (cell id) using the coordinate from @board.cells ['A1']